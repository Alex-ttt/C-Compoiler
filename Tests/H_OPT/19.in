void main(){
	if(4){
		int r;
		r = 5;
	}

}