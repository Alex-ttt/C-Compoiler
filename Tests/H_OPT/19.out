 Symbols Table:
   main  void function main(
  )
    {
      Statements:
      if

        4

      then

        {
          Statements:
            r
          =
            5
          Symbols Table:
                      r int
        }
    }
